backend/wmg/api/v2.py

```diff
@@ -51,7 +57,12 @@ def query():
             explicit_snapshot_id_to_load=WMG_API_FORCE_LOAD_SNAPSHOT_ID,
         )
 
-        q = WmgQuery(snapshot)
+        cube_query_params = WmgCubeQueryParams(
```

Comment:
do we need to make the equivalent changes to the v1 API?

Reply:
`V1` will be deprecated. I have already removed unit tests for `V1`. There is another ticket to deprecate V1: https://github.com/chanzuckerberg/single-cell-data-portal/issues/5361